deploy/olm-catalog/compliance-operator/manifests/compliance-operator.clusterserviceversion.yaml

```diff
@@ -1278,14 +1278,16 @@ spec:
                     fieldRef:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
-                  value: compliance-operator
+                  valueFrom:
```

Comment:
@JAORMX this change comes from commit eb500977143174759bf677508da36a566e4a4c70 (I think?) but what confuses me is that the change was included in the previous release, not this one. When you did that release, did you hand-edit the CSV to exclude this change? Or do you know why the change appears in this release and wasn't already in the previous one? I'm concerned about upgrading, because locally I can't upgrade from a previous version just by `oc apply -f /path/to/deployment.yaml`:

```
The Deployment "compliance-operator" is invalid: spec.template.spec.containers[0].env[2].valueFrom: Invalid value: "": may not be specified when `value` is not empty
```

Reply:
I didn't hand-edit the csv, just used our normal `generate` target

Reply:
weird. But we're test the upgrades anyway, worst case we revert.